lemminx-liberty/src/test/java/io/openliberty/LibertyHoverTest.java

```diff
@@ -255,23 +256,20 @@ public void testXSDSchemaHoverWithInvalidLocale() throws BadLocationException, I
                         "       </featureManager>", //
                         "</server>" //
                 );
-                // should default back to 25.0.0.6
+                // should default back to 25.0.0.8
                 XMLAssert.assertHover(serverXML, serverXMLURI, "Defines how the server loads features." + //
                                 System.lineSeparator() + System.lineSeparator() + //
                                 "Source: [open_liberty_schema-25.0.0.8.xsd](" + serverXSDURI + ")", //
```

Comment:
Had one more thought....you could use the constant for the default version here so that the test case will not need to be updated...it would always verify that the default is used.

Reply:
Sorry, but I mistook this test case to be testing the default (which is actually 25.0.0.6). Existing test cases that are actually testing the default could use the DEFAULT_LIBERTY_VERSION from LibertyConstants. This test is just defaulting to the latest which will keep changing. Not much you can do there.